docs/admin-api-endpoints-authorization.md

```diff
@@ -0,0 +1,4 @@
+# Admin API endpoints authorization
+Admin API endpoints authorization is granted by membership to rover groups that represent the -read/write/full roles per environment. More information can be found [here]() // TODO add link for the merged architecture repo ADR.
+
+Configuration of the roles for development purposes can be found [here](../config/admin-authz-configuration.yaml). When deploying kas-fleet-manager to an OSD cluster, `ADMIN_AUTHZ_CONFIG` can be provided to control the authorization for the admin API endpoints.
```

Comment:
Let's also include these https://github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/blob/fd1700778497d649ae4943d8d732c664f6361e28/templates/service-template.yml#L419-L429 in the documentation